api/src/main/java/com/datatorrent/api/StreamingApplication.java

```diff
@@ -47,6 +47,12 @@
   String ENVIRONMENT = DT_PREFIX + "environment";
 
   /**
+   * Use this config variable to to add any external library to classpath for the DAG.
+   * Value of this variable is same as StramAppLauncher.LIBJARS_CONF_KEY_NAME.
+   */
+  String LIBJARS_CONF_KEY_NAME = "tmplibjars";
```

Comment:
The reason why it's added here is constant for tmplibjars is present in StramLauncher which is part of apex-engine and if this variable is to be used in malhar, apex-engine won't be added as dependency to malhar.